core/store/src/config.rs

```diff
@@ -65,6 +65,8 @@ pub struct StoreConfig {
     pub load_mem_tries_for_shards: Vec<ShardUId>,
     /// If true, load mem tries for all shards; this has priority over `load_mem_tries_for_shards`.
     pub load_mem_tries_for_all_shards: bool,
+    /// If true, load mem trie for each shard being tracked.
+    pub enable_mem_trie_for_tracked_shards: bool,
```

Comment:
```suggestion
    pub load_mem_trie_for_tracked_shards: bool,
```
For consistency with other parameters. Or replace with LoadMemTriesMode uniting all that.